src/profile-logic/marker-data.js

```diff
@@ -471,7 +699,7 @@ export function mergeStartAndEndNetworkMarker(markers: Marker[]): Marker[] {
               : [markerNext, marker];
           const mergedMarker = {
             data: endMarker.data,
-            dur: endMarker.dur,
+            dur: startMarker.dur + endMarker.dur,
```

Comment:
I needed it in my tests so I did it here. But I plan to finish https://github.com/devtools-html/perf.html/pull/1719 instead, likely this afternoon, so that it won't be in this PR.

Reply:
done in #1752 :)